docs/add-ins/addin-tutorial.md

```diff
@@ -35,18 +35,14 @@ You will also need Outlook 2016 connected to an Office 365 account, Outlook.com
 
 Open your command prompt/shell in an empty directory. Enter `yo office` and follow the prompts.
 
-- **Would you like to create a new subfolder for your project?:** `No`
-- **What would you want to name your add-in?:** `Git the Gist`
+- **Choose a project type:** `Jquery`
+- **Choose a script type:** `Javascript`
+- **What do you want to name your add-in?:** `My Office Add-in`
```

Comment:
Could you keep the name Git the Gist here?

Reply:
Good catch! Updated.